dev/run-tests.py

```diff
@@ -283,8 +283,8 @@ def get_hadoop_profiles(hadoop_version):
     """
 
     sbt_maven_hadoop_profiles = {
-        "hadoop2.7": ["-Phadoop-2.7"],
-        "hadoop3.2": ["-Phadoop-3.2"],
+        "hadoop2.7": ["-Phadoop-2.7", "-Phive-1.2"],
+        "hadoop3.2": ["-Phadoop-3.2", "-Phive-2.3"],
```

Comment:
Hm, @dongjoon-hyun, can you create JIRAs to follow up? Seems we will change `-Phive-2.3` as a default profile as [discussed in the mailing list](http://apache-spark-developers-list.1001551.n3.nabble.com/Removing-the-usage-of-forked-hive-in-Apache-Spark-3-0-SPARK-20202-td28347.html).

1. Verify *Hadoop 2.7 and Hive 2.3 combination in both JDK 8 and [JDK 11 (WIP)](https://github.com/apache/spark/pull/26533) as it will be the default
2. Setting a Jenkins job for `-Phadoop-2.7 -Phive-2.3`.
3. We will need to be able to configure Hive version and also Hadoop version in the PR builder.
4. Change the default profile to Hive 2.3.
5. https://github.com/apache/spark/pull/26619#issuecomment-556926064
6. Release script updates.
7. ... (other more .. ?)

*Hadoop 2 will be default at this moment. it's [being discussed in the mailing list](http://apache-spark-developers-list.1001551.n3.nabble.com/Use-Hadoop-3-2-as-a-default-Hadoop-profile-in-3-0-0-td28198.html)

Many things are going on so it looks very easy to lose the track.

Reply:
Sure!

Reply:
BTW, the thread is independent from JDK11 or Hadoop 3. For the following, only JDK8 is the target of the follow-up of this PR. And, for JDK11, I believe @wangyum will handle it his on-going work.
> Verify *Hadoop 2.7 and Hive 2.3 combination in both JDK 8 and JDK 11 (WIP) as it will be the default

For (4), `Hive 2.3` is already default for `Hadoop-3`. And, this PR makes `Hive 2.3` as a default in the pom files. So, I guess the remains of (4) is equal to (2) and (3).

Reply:
The follow-up issues are created and mentioned in the PR description, @HyukjinKwon .

Reply:
Ah, sure, thanks for clarification. @wangyum, can you verify the new JDK 11 combination and take a following action?

Reply:
@HyukjinKwon I will do it later.